common/solar_system_initializers/hfe_initializer.txt

```diff
@@ -2,11 +2,14 @@
 # egypt
 hfe_initializer_egypt_main = {
 	prevent_anomalies = yes
+	usage = fallen_empire_init
 	flags = {
 		hfe_egypt_0
 	}
 	name = "Home of Light"
-	class = rl_standard_stars
+	class = sc_m
+	spawn_chance = 0 #Only by Event
```

Comment:
Make no sense

Reply:
The Ju Ahets said that they literally lit a new sun, so their star is going to be artificial. In RL, Red Dwarf stars are the most stable and common stars and will last trillions of years. They were thinking about the rest of their existence.